### ISSUE TITLE
ModExp 8192-bit limit not enforced

### ISSUE THREAD

**Issue by praetoriansentry** [0xPolygon/cdk-erigon#1385]
The prover / executor will have a limit of 8192 bits for any particular argument into. Right now it seems like cdk-erigon doesn't enforce a similar limitation so there is potential for mismatches.  E.g. the arguments below have a `base` value that's 2048 bytes and `mod` value that's also `2048` which is much larger that the 8192 bit limit:

```
Bsize: 0000000000000000000000000000000000000000000000000000000000000800
Esize: 0000000000000000000000000000000000000000000000000000000000000008
Msize: 0000000000000000000000000000000000000000000000000000000000000800
B:     443d41801d15ec86a7d41b75c2d198392e19eb809b65ad5e4de212311511bc79ffe9c7ab14bf8cf6e9bb2a46520c770b571f67f3cc4eddb7f293444248187a6b0bc62a41fc1a185d61fb825ade3a8e7fa117264b8b7366015b67bf382e239261cead61c2bf306596c85af4d0ed29ce42084fdc6e54d544a6c6e70be5f441404e406188fa2a7afc9a3d499bcaf1a5c94ffccc5035118b7a3189be57c620bc82c1e9a04d1f2d26323de700bdba8de8de7158730ceb7d8d24b5fc22fffb53a10e4f1240f310714b1ecbcd33556729f72c6439cf57b69885afa822c548342f96cce0947e43d81692bbce65b7966c0d9b87b5bafd6a45a3eccf5546bcb1326f77c52a174c395ca0b9cf27c06f16d0f294cea57891ea8aa4df8d6a5044156c82882bd5ad6282d2a8f3d5170cb920ba5e4c1f585aef5e881d12471fb21f750bfa108dedd6f53dedb7051f3fceb3f87e377555005f5f0ed74645b1bb3e9fc51fd14de3110836c094ed3264eb72e8cdf9de46c8321be0ae04ab7519ab0fc8f4683c58ae0c87aede51dd0c1fb048b5393a722a36f6f864022bd648e2aeb0e6095d5321275509b230f81dc2901c2eff36ba6e9c73c1d04b1e888a467a6c5c081ecf772c4ac6e0b5c631331650e63082401ab39370927c69f96ee4d9010ceb84c024b6863fe9cc6b279b25560b536f0783171d91ef1e8f18a1e2de0f2b44165c01fa543303064d87a7d20c5bcb208341eb0274ba48fddfd1ee27778b6dfd2199518c3f16a9ccd8fcc3762d35892c6f9084897a535fa316cc33ab14a18d6ce6cbd931530c0d75eef50ee58ef5e645845f214c8d90b4d4915e704c8ac34495d4335b84f89364c7c7a1404b4bf91f7a73eac52d345460e80ce6929353c5d2246fbd07dc8da2b913f64d6ad6f5686ee2a2351cb0c9e434379b2174aec2c4d719b766707c016de37f92d4362bceebe4f178dcb176f7fd40a2931f2296b8e32f8e7d7ae0cd1b99cc4050cbab1b025b40fe53f2061d3187a6de2db75856123694715a23329548f3619c685d95e6f9b442d05393239214b9fe5396b158bed11224fd8998ab0bd284719d18f3f4d1b084575c330abdd5097af25811c5c2d67644a96e9d73e93be9f3c53b23f238c74bfaba838c8cb16e993dac858f329865ae879bdf3f38e4dc4d5e5cb21d268d5984809f97edece6b2c35c42405cde51172e8c13f75e62191db8a6b2d72e20dc87e6152f1935ece923b818445282c7c1544926de366c8f82e4990e199cc10c519d70e81ef5675840cd7a867995624ab670fb285cd970820933f15353b0001e6fe85cf447c864d5e604a9fe7f621d3e416fdb1054289cb3b4cef79f95221fa50e8778a33e256f80285c928610cd1e77243a5f916a62b5462be1515ac230a3a045e8de61cd15c2cf92814584650dd204a820de40f471327e14cddf54b9be0c85a5761181b295e355bc250efbf1e5449a4cae2f1906cceabc7562a3761f41a70f3bea37e1ee2ef80974d1a1464a7a7ca3ce69637b5e11db9b9ad08c4857d0bb1a2dd8b5595e4de9bb5fdf1a8777c8de8e4a158d60fe6a16504613ce051c060f7acadaf1665c49474c438b9f51c67acf15d15a4bdbc5eb8c7bf6327da8374e4fbfd2991e9fb8e9cd3fd3b8add03ad247b499c9ca4faa4a476e65ca1d76de7fe61dfbd5a1226fba7fed5de6d138cc842f826d43e2f8dfdbe3d7ae7ae70663f6223cc1ebe377efa05246a8aab61ee37f95eb45e0cca5326f33b02ea8eadeeba200dc74d3ab71b1b75525d3f2d726602cb0a9a1d330cfa9fc8b8c71b2f2223cc3e6c7ca37649ce624483692cf2d51269e16d67c4e45062a07ffb68b3b638c18f655080df5bee78c877203dc7a31220a29ef5bf6012fa3f33ae1f9d5d5b8183b1550e8b97ba54eb741295ff495c02267450efdff2afc63d07bd9ce249ccf5e317971052eb630f6dde1e0c1a34c8170defe44ac82716bc49d62afe96d0330f6e4c461e5097bb710e72bb1ea7d8d85c4f45fe76e38903e2b3b48bce62b3158d55e52bfa2ef7bdc5860af47e756267c01238ce2d95e1d8120b59991d0752e8bc643979cba20e77770faba547db801990c11338c2c93f6daf85c8400609ae2d10b90c33eba840394badd1539557dd180e3265f8646aafcc3932aaeda61c6ac3d3bf81fd3cc11739894cf5cfd15bd1c3523d98b4bb7329427b8c500a7746e34849d56b0c6b7a6756b61b4095a2c69c841d15fc9a798469db3614a98a735f63e6e1f5c850e1aaa1e6b35451ad2f2a184c8323d33a858463bb8ba96afda8e62525262aeaea34840e520ffd6a501161ec86e81a8df34106b2d7edff47669159ccb50f581521986458528a5abc041074e4ae121fd5185365c7c20d0eb879f591f500730756dac42488a14584c2295aa48a223908072342aa6b4c2c6d40aac6c519d0ce19f850387a42350cc155ddab8245ba1ae0331e70f661c95913b1d03f404d77292dc79f81358a626d90c1ca52be18411a1f073bf9e5bec73aea76d05436d4f5c687a047381525c657e04eb73af20f996b11d1a399163ae5c54e6c884a36cfbd8d6fcb4c9034d83fcbb231eb9f8bcc19b763a35e07e4d3a0f4464e78ec693f084a4f5cb5782cefcc49fabaaf3105eaa0b07c03b0445cb5d153233e268dc4b2d271d1a7289f806b9e6081ee5b6cd1ca68f3519bfb1cc54b1bdaa7de5f210befee182f23cfaa6552f05b389093d7dfff7882a14c5870c044ba6fc881f536a28e11081a7d370bf50fed8e8c0791dd95a949fe380eab696bcafadec8ba32b244708dfec32f780c87eb91ce79309dab7e73bddd089ca325e8ba1d63097ddb5908e988e1da0caa172de24bb2193cdeee0853a17f78133a94b9f59dc05da7bef27547f810ce4bd
E:     2169a2e133d2dcc1
M:     13bb5559c3aef7af8955dc6e21314a5bf6546181a1d2f9599a911426cbd1b1b222f9881062123774d3ca8bf6faa99ffe74ff746350933e2045a75fe934a989dcb013f2e8d06f7d20f81f277a856860b6342d0cb887bf9d37ab1f60bb3f439678403554bbb7144fce2c22ac40e9c4482d9d50627f8f10787d2021154d507fe73bacfc1a4e17dc53cf8863515c1fa8d85298a45a167d4f16cc21ac53189a72998c718a5adce8032193f8a60d8b9bb93f00d0dd275ca1b30142af7e3ac8170a18697d19d35a53790e59a72f02b2397e3b8a692b58bc945b2f5da35c0e4d5272e38532eff1abd770cfae373f8231b5ebc6390e5e2b2ad9dd11caaff331a62a32c0d5098a917589c63a1fc9ec717162bf72af16b3ee97fb4bb1021b8dada76949b36f6e1c2177c966f0395bca75d4a0b1abff74bf1501f6cb608e0767ad60764061bfb231522bd5da322035fccbd72c61578bf14b3edd25011ee0c7668988e94a2e25ef5094311fbc131bc6cff84042dff1285e7df02120a18345a4c3c9c4208f9b343542837e3b3df491272c0439dfa8a578352c504fb8580c5e69ff6f0e280858e92a72e79e6695fb06bc8c16b6dd6c5c859e238e35b95bf2bc3cd4f04f9f4d909f7cda3ac699e7287ee9d703233b170343ecf5e61e4c9da4338d7316b779f42b67d2895d3c0137d1f1e4e23e85054dba3f5b7768b88dfb9898d7c347e1f2c59e4e35d4d3a0cdb1a4e32bc1317e9cc4329228f25cbf9ea63f05974d1d7caa1e5ff3ee2afaf022193e796ae5a50c44ab92104cf926c13204b345ccf6b72a6f9ccc3396984fd1afd9ef694df6d84a5c8ce7b9c06d70bffd586ef0f06ba62663b1ce69697f03953a8387f79dc6d5e34059d4e4487f05735ebe1fe09e0a12ca475fa4da993601da39d177086ad4220fc4004cab76ec989349c6512107a8df49711c88a404379685fe0a98883a8e1e5051100362611cdbc06e86b9d15dc6f6bd6af64f2f3175803c47a164d3021105ece016b8d06ef74d081d3381a86ecff69360ee93b5e6a609528b157dfc27a4a3754dba12b8741ccdb17aa1e993bf29a69470c2242c80f09e5e54936ec0cb0ebeefc895bb15220b61158c77f94b5e3337a5f4948bd086a5da7d0e4f667cf1cf54e8eb259aa62f593280b0038d4b179ff5793961c7e00512f20e17a42a1a44f3b1391ac6b9ee001280248f60d68cf9b1a5cc4cbaef9308fcd8e2749e1ff482f2c6b42eb5795be376ec1818d1cd0f1ed9aaaf5d80e3d106c53ff33509393827bac6dabaeaebabbadab94150dd9888e6f9c718f828d32ed7aa7cc9b9dc50257ee4a79d2e8175e4d20834e46d924e21b280c926074dfaed66ab33c06dcd1e07f469165621b6097606429838ae24cd568bca9e67933a9b045c567801d805464cc146d45f96a450768b889d6cf749defd01b34a1085c3da27ff00ce9e8a6bf3615bc3f72c921d20e581eb3d4fa4898c568b62b2f02eb2fce1fe6641fe3e506ac10b92fe6343761615b7834d70178fda2b238d48a7184b72243f48aa80fdfa318f15ef5df1e696606ff5a4fa3eb88b33460e07e609ce371a37bb7d4d063eddae2b249cd865d4aefcadf77a628c8e735820606a9fad9773bcb1f66af359b01ee808550c8c641a0068fb0b4da2073b18a39f28ce109ef1bfcb0b73baeaff2440d5294b67f914e1ddef4c1c2604ad216df7f50fa28a880552bcf3469a32a10ca20e43b2b6d32b66ed31ad1af71ef6f6db14355dd5115e61a6a26bc584f01947e8a614147be13da4a840fff81a946dd74b73d951ff165c01f82ce05d894571ba670b77575d8348508d5d61bf8c62dc5c627296aacf169eafdc8ac85100d9bddc8fb6313e6024bfd36ad391875cbb9d034ca2e2e487fc001092bb67d5ed33f9e2aedb469bbac69665c733c0f590f439cdb05d2819110794667fb32bce09faa7b1d987c440c1d7b8f0d565c313cfbe06e01139d3481b04d8687d2f9086ca2eaaf8f6963e013fbe6e47583cf8bfa0e1c7d63db5babad1358f26bfe3eb40d6ec86bd1189e77a7fbaec68874d98e4ac31b7c1cfceb85bd145a343cbca581eecec6bbf1705cec6a2e3b3ac66bc924980a1663792e98b98637431ebd977d70da14e67eb4f2f3d89b55af504f3436abf1fd8da9668cbac125ac666195a150eae228c05ff0a7e646d81dca78c7975f283f25fc6059a53c623451cc7074b12247fb0c529b8b1f27bc4d203253d0345f6de6b5cb7068089ef1caa9a25ce3621b0bb8d001777c8a29e713508af8a0523c65cf684abee4bdf6152c10389d855016d7def83752b524dc2ec5ea02fb9e74bd7f7baace83563d556a6359dcca3dd86048e0ed65dc0451ef35fa3e0a2363d05a1f95d0b8545a0961406286d5fa4149d247124fe1566946846a168347afd050745cf26403dd7c87e8d5921e7cf43a269cfa4dd9f52e93502878fad5d09b51266b6c1650247bbc9b7715cc4ae0b2ee2629e91b3da76b8a07d08a60bb42596379d2e1bda9f53152fa8869c44701521725a5ca495c581ee1791747bc02aafc9ca0c6d8b0277ea5ce30039522910030e386280ae7a1652d82a185bbc18cdc8a4398664ac3b91f640c3a8bc8cf5117790920ac9427e0143fffa908a866c9b991201583c99dba975c46fd4e0c0670c38af24253af184acfdff2b62f193f7dbbbae0ad9f930881a2d0926f7485a888106dfe070a22194bee74bba527f19bf4920617e288d03462c72ef04b1d0b2661a45de5948b75ea89e9faff6a56faf80910d6e651e7ba66bd9ab83687816a940ff1aa7eb6c012d4e782ad5ce0ed4c209c7293314161dbd1b34ded8bf69ad3483b67cdb915b567d1a4ce339aa7e75614efbdf6a3a195b59a79f813f343e3bd266efc21aea974d
```

We can execute this with a call like:

```bash
cast call --rpc-url http://127.0.0.1:33162 0x0000000000000000000000000000000000000005 000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000800443d41801d15ec86a7d41b75c2d198392e19eb809b65ad5e4de212311511bc79ffe9c7ab14bf8cf6e9bb2a46520c770b571f67f3cc4eddb7f293444248187a6b0bc62a41fc1a185d61fb825ade3a8e7fa117264b8b7366015b67bf382e239261cead61c2bf306596c85af4d0ed29ce42084fdc6e54d544a6c6e70be5f441404e406188fa2a7afc9a3d499bcaf1a5c94ffccc5035118b7a3189be57c620bc82c1e9a04d1f2d26323de700bdba8de8de7158730ceb7d8d24b5fc22fffb53a10e4f1240f310714b1ecbcd33556729f72c6439cf57b69885afa822c548342f96cce0947e43d81692bbce65b7966c0d9b87b5bafd6a45a3eccf5546bcb1326f77c52a174c395ca0b9cf27c06f16d0f294cea57891ea8aa4df8d6a5044156c82882bd5ad6282d2a8f3d5170cb920ba5e4c1f585aef5e881d12471fb21f750bfa108dedd6f53dedb7051f3fceb3f87e377555005f5f0ed74645b1bb3e9fc51fd14de3110836c094ed3264eb72e8cdf9de46c8321be0ae04ab7519ab0fc8f4683c58ae0c87aede51dd0c1fb048b5393a722a36f6f864022bd648e2aeb0e6095d5321275509b230f81dc2901c2eff36ba6e9c73c1d04b1e888a467a6c5c081ecf772c4ac6e0b5c631331650e63082401ab39370927c69f96ee4d9010ceb84c024b6863fe9cc6b279b25560b536f0783171d91ef1e8f18a1e2de0f2b44165c01fa543303064d87a7d20c5bcb208341eb0274ba48fddfd1ee27778b6dfd2199518c3f16a9ccd8fcc3762d35892c6f9084897a535fa316cc33ab14a18d6ce6cbd931530c0d75eef50ee58ef5e645845f214c8d90b4d4915e704c8ac34495d4335b84f89364c7c7a1404b4bf91f7a73eac52d345460e80ce6929353c5d2246fbd07dc8da2b913f64d6ad6f5686ee2a2351cb0c9e434379b2174aec2c4d719b766707c016de37f92d4362bceebe4f178dcb176f7fd40a2931f2296b8e32f8e7d7ae0cd1b99cc4050cbab1b025b40fe53f2061d3187a6de2db75856123694715a23329548f3619c685d95e6f9b442d05393239214b9fe5396b158bed11224fd8998ab0bd284719d18f3f4d1b084575c330abdd5097af25811c5c2d67644a96e9d73e93be9f3c53b23f238c74bfaba838c8cb16e993dac858f329865ae879bdf3f38e4dc4d5e5cb21d268d5984809f97edece6b2c35c42405cde51172e8c13f75e62191db8a6b2d72e20dc87e6152f1935ece923b818445282c7c1544926de366c8f82e4990e199cc10c519d70e81ef5675840cd7a867995624ab670fb285cd970820933f15353b0001e6fe85cf447c864d5e604a9fe7f621d3e416fdb1054289cb3b4cef79f95221fa50e8778a33e256f80285c928610cd1e77243a5f916a62b5462be1515ac230a3a045e8de61cd15c2cf92814584650dd204a820de40f471327e14cddf54b9be0c85a5761181b295e355bc250efbf1e5449a4cae2f1906cceabc7562a3761f41a70f3bea37e1ee2ef80974d1a1464a7a7ca3ce69637b5e11db9b9ad08c4857d0bb1a2dd8b5595e4de9bb5fdf1a8777c8de8e4a158d60fe6a16504613ce051c060f7acadaf1665c49474c438b9f51c67acf15d15a4bdbc5eb8c7bf6327da8374e4fbfd2991e9fb8e9cd3fd3b8add03ad247b499c9ca4faa4a476e65ca1d76de7fe61dfbd5a1226fba7fed5de6d138cc842f826d43e2f8dfdbe3d7ae7ae70663f6223cc1ebe377efa05246a8aab61ee37f95eb45e0cca5326f33b02ea8eadeeba200dc74d3ab71b1b75525d3f2d726602cb0a9a1d330cfa9fc8b8c71b2f2223cc3e6c7ca37649ce624483692cf2d51269e16d67c4e45062a07ffb68b3b638c18f655080df5bee78c877203dc7a31220a29ef5bf6012fa3f33ae1f9d5d5b8183b1550e8b97ba54eb741295ff495c02267450efdff2afc63d07bd9ce249ccf5e317971052eb630f6dde1e0c1a34c8170defe44ac82716bc49d62afe96d0330f6e4c461e5097bb710e72bb1ea7d8d85c4f45fe76e38903e2b3b48bce62b3158d55e52bfa2ef7bdc5860af47e756267c01238ce2d95e1d8120b59991d0752e8bc643979cba20e77770faba547db801990c11338c2c93f6daf85c8400609ae2d10b90c33eba840394badd1539557dd180e3265f8646aafcc3932aaeda61c6ac3d3bf81fd3cc11739894cf5cfd15bd1c3523d98b4bb7329427b8c500a7746e34849d56b0c6b7a6756b61b4095a2c69c841d15fc9a798469db3614a98a735f63e6e1f5c850e1aaa1e6b35451ad2f2a184c8323d33a858463bb8ba96afda8e62525262aeaea34840e520ffd6a501161ec86e81a8df34106b2d7edff47669159ccb50f581521986458528a5abc041074e4ae121fd5185365c7c20d0eb879f591f500730756dac42488a14584c2295aa48a223908072342aa6b4c2c6d40aac6c519d0ce19f850387a42350cc155ddab8245ba1ae0331e70f661c95913b1d03f404d77292dc79f81358a626d90c1ca52be18411a1f073bf9e5bec73aea76d05436d4f5c687a047381525c657e04eb73af20f996b11d1a399163ae5c54e6c884a36cfbd8d6fcb4c9034d83fcbb231eb9f8bcc19b763a35e07e4d3a0f4464e78ec693f084a4f5cb5782cefcc49fabaaf3105eaa0b07c03b0445cb5d153233e268dc4b2d271d1a7289f806b9e6081ee5b6cd1ca68f3519bfb1cc54b1bdaa7de5f210befee182f23cfaa6552f05b389093d7dfff7882a14c5870c044ba6fc881f536a28e11081a7d370bf50fed8e8c0791dd95a949fe380eab696bcafadec8ba32b244708dfec32f780c87eb91ce79309dab7e73bddd089ca325e8ba1d63097ddb5908e988e1da0caa172de24bb2193cdeee0853a17f78133a94b9f59dc05da7bef27547f810ce4bd2169a2e133d2dcc113bb5559c3aef7af8955dc6e21314a5bf6546181a1d2f9599a911426cbd1b1b222f9881062123774d3ca8bf6faa99ffe74ff746350933e2045a75fe934a989dcb013f2e8d06f7d20f81f277a856860b6342d0cb887bf9d37ab1f60bb3f439678403554bbb7144fce2c22ac40e9c4482d9d50627f8f10787d2021154d507fe73bacfc1a4e17dc53cf8863515c1fa8d85298a45a167d4f16cc21ac53189a72998c718a5adce8032193f8a60d8b9bb93f00d0dd275ca1b30142af7e3ac8170a18697d19d35a53790e59a72f02b2397e3b8a692b58bc945b2f5da35c0e4d5272e38532eff1abd770cfae373f8231b5ebc6390e5e2b2ad9dd11caaff331a62a32c0d5098a917589c63a1fc9ec717162bf72af16b3ee97fb4bb1021b8dada76949b36f6e1c2177c966f0395bca75d4a0b1abff74bf1501f6cb608e0767ad60764061bfb231522bd5da322035fccbd72c61578bf14b3edd25011ee0c7668988e94a2e25ef5094311fbc131bc6cff84042dff1285e7df02120a18345a4c3c9c4208f9b343542837e3b3df491272c0439dfa8a578352c504fb8580c5e69ff6f0e280858e92a72e79e6695fb06bc8c16b6dd6c5c859e238e35b95bf2bc3cd4f04f9f4d909f7cda3ac699e7287ee9d703233b170343ecf5e61e4c9da4338d7316b779f42b67d2895d3c0137d1f1e4e23e85054dba3f5b7768b88dfb9898d7c347e1f2c59e4e35d4d3a0cdb1a4e32bc1317e9cc4329228f25cbf9ea63f05974d1d7caa1e5ff3ee2afaf022193e796ae5a50c44ab92104cf926c13204b345ccf6b72a6f9ccc3396984fd1afd9ef694df6d84a5c8ce7b9c06d70bffd586ef0f06ba62663b1ce69697f03953a8387f79dc6d5e34059d4e4487f05735ebe1fe09e0a12ca475fa4da993601da39d177086ad4220fc4004cab76ec989349c6512107a8df49711c88a404379685fe0a98883a8e1e5051100362611cdbc06e86b9d15dc6f6bd6af64f2f3175803c47a164d3021105ece016b8d06ef74d081d3381a86ecff69360ee93b5e6a609528b157dfc27a4a3754dba12b8741ccdb17aa1e993bf29a69470c2242c80f09e5e54936ec0cb0ebeefc895bb15220b61158c77f94b5e3337a5f4948bd086a5da7d0e4f667cf1cf54e8eb259aa62f593280b0038d4b179ff5793961c7e00512f20e17a42a1a44f3b1391ac6b9ee001280248f60d68cf9b1a5cc4cbaef9308fcd8e2749e1ff482f2c6b42eb5795be376ec1818d1cd0f1ed9aaaf5d80e3d106c53ff33509393827bac6dabaeaebabbadab94150dd9888e6f9c718f828d32ed7aa7cc9b9dc50257ee4a79d2e8175e4d20834e46d924e21b280c926074dfaed66ab33c06dcd1e07f469165621b6097606429838ae24cd568bca9e67933a9b045c567801d805464cc146d45f96a450768b889d6cf749defd01b34a1085c3da27ff00ce9e8a6bf3615bc3f72c921d20e581eb3d4fa4898c568b62b2f02eb2fce1fe6641fe3e506ac10b92fe6343761615b7834d70178fda2b238d48a7184b72243f48aa80fdfa318f15ef5df1e696606ff5a4fa3eb88b33460e07e609ce371a37bb7d4d063eddae2b249cd865d4aefcadf77a628c8e735820606a9fad9773bcb1f66af359b01ee808550c8c641a0068fb0b4da2073b18a39f28ce109ef1bfcb0b73baeaff2440d5294b67f914e1ddef4c1c2604ad216df7f50fa28a880552bcf3469a32a10ca20e43b2b6d32b66ed31ad1af71ef6f6db14355dd5115e61a6a26bc584f01947e8a614147be13da4a840fff81a946dd74b73d951ff165c01f82ce05d894571ba670b77575d8348508d5d61bf8c62dc5c627296aacf169eafdc8ac85100d9bddc8fb6313e6024bfd36ad391875cbb9d034ca2e2e487fc001092bb67d5ed33f9e2aedb469bbac69665c733c0f590f439cdb05d2819110794667fb32bce09faa7b1d987c440c1d7b8f0d565c313cfbe06e01139d3481b04d8687d2f9086ca2eaaf8f6963e013fbe6e47583cf8bfa0e1c7d63db5babad1358f26bfe3eb40d6ec86bd1189e77a7fbaec68874d98e4ac31b7c1cfceb85bd145a343cbca581eecec6bbf1705cec6a2e3b3ac66bc924980a1663792e98b98637431ebd977d70da14e67eb4f2f3d89b55af504f3436abf1fd8da9668cbac125ac666195a150eae228c05ff0a7e646d81dca78c7975f283f25fc6059a53c623451cc7074b12247fb0c529b8b1f27bc4d203253d0345f6de6b5cb7068089ef1caa9a25ce3621b0bb8d001777c8a29e713508af8a0523c65cf684abee4bdf6152c10389d855016d7def83752b524dc2ec5ea02fb9e74bd7f7baace83563d556a6359dcca3dd86048e0ed65dc0451ef35fa3e0a2363d05a1f95d0b8545a0961406286d5fa4149d247124fe1566946846a168347afd050745cf26403dd7c87e8d5921e7cf43a269cfa4dd9f52e93502878fad5d09b51266b6c1650247bbc9b7715cc4ae0b2ee2629e91b3da76b8a07d08a60bb42596379d2e1bda9f53152fa8869c44701521725a5ca495c581ee1791747bc02aafc9ca0c6d8b0277ea5ce30039522910030e386280ae7a1652d82a185bbc18cdc8a4398664ac3b91f640c3a8bc8cf5117790920ac9427e0143fffa908a866c9b991201583c99dba975c46fd4e0c0670c38af24253af184acfdff2b62f193f7dbbbae0ad9f930881a2d0926f7485a888106dfe070a22194bee74bba527f19bf4920617e288d03462c72ef04b1d0b2661a45de5948b75ea89e9faff6a56faf80910d6e651e7ba66bd9ab83687816a940ff1aa7eb6c012d4e782ad5ce0ed4c209c7293314161dbd1b34ded8bf69ad3483b67cdb915b567d1a4ce339aa7e75614efbdf6a3a195b59a79f813f343e3bd266efc21aea974d
```

The expected output is:
```
0969af9b5dd2c76128fdbc9a0f4444f918113e244df7096b774115c7edb7b1ad81527154120a0bf88aee473ff9889bea4b2455b772e66c0b8b6dc466f60854d5eeb2e00bc309904189e454f00573496fe88f783471f1eb6004ff51b44bc11d0bf662b408a2754619d2eed575e4ffa8c8b37b5201f76e16c89c4cf7b6a92cb6f9132fe877133c8ac8d3afdcda0f286c9f8fd80874fb237a1e40e313fec0d39ffe678a60858575c29acd6e4dfe59e9e3a87c7ff1dd6126399323635795a08b59061ff7cb2d2cee425d26bf94b99d35d22c7ce3fefc1bbac112f52334a8a9915cd82cbd32b4c9d897f8d29776149cabb25cb11b989458bcb99fd69830864c7c34523e594e26266804b6750aed21984e99e4d74dcbc7087c6fd090a7d1f3a2cdd0725101aaa868fa15a47a73eb414e3cca603b0d0f84e6bd3b2ad756e432684cf96cb33be5ee96bd175b711c8720d963d0b7f4a34ed1270f9d37124b3ccd3ef7df5ca7b78e0caf29c81ad58afb1d223d1eabeab9d6190cdd042d9728c865661246e139eb8ede2da9c539e39e2f0fed92c7a13761f62e439dcb09862301f098d1cd17fb0c1977c43a9c1a16b2d934e853b090bc6fb9cb93a8641d7a5d0cec6d7636e71f898d5b5bbaf0f877cce037588c53d65270f052fa498ec5130ec82f328ee712a17bdc39dc60e30700011cdf7950a806974ec6f808e1cce7dc9100cddf94140c544822e09bc4cf9f3fcb89d7ead3ff94f68b09a51a2926258aba72870ab30ca0f6437c29d9fdf2b8365f1fec8d177592e4d877105686dedbf9060f5d7ebfffda003802e52e5b43bc75cbeda8168783a107784a8805393aa7aefcd50acc6e52881214f365172d7fcc52b871c4ba9f65fbce4b8b12866f6bd78d9dddd048ba5e2408fed72ac07dbe86cf2fd6cd588a7035233772f425715b6b8a292fbc8b12e411c9b162f68f72c97ead47467a3b7a5afd088511be4c41a541ef7bacc616ff4592a4b8069af8898abf26dd0deea1a0c402983e9fb8c443aed32f92fa1888d75657a996d86c3772d860176c561d5917806f033c13a04493c8b4a6c6cf17235cd6ba0db2e56d0e6436826d233e07846b1cacceaf5248d66c65bb135448f85dbb42b66f32ff84003c92fd3d965061e5bd830734a79640b8c4c8f644169cd3fa014a383816dbd87bfe75fae866039d2b6a3ab9ea4779ad11537ff385993130f9887b851f0840c867c749c47e0ff6daea6eae253c64abd8079e1a8b1e96f32e6c0b78fb041d7e5a9dbc15ace1d5d03dad901e8963feefb2a383091080ffc432a56c91dd47c616b52b9c520a9dad1bc7cee5842ab4a84ac175160581e9d5b693364bfffeaf68da7c75e673623349d0c76e30e41203a46a6ec355ce63f8ae9567c9aef92511d24820d5233587147c329eae7639707ef45eadea8e39426b2cdd0b715c6047d1df179eb2210a74d33493618788973cacac77dbc9a51fb48c8aac4b36f236f2d18f419ab12e1d7e32700d3dd1a42d46c2179d7acae4c968438700130351b8a60c64d9c6dc2e52c2b450847d01908282bcb886602240f0ecc8a2d70bb22189d269586895c2e787b92da26d7cd2f5029439a4bd680d1d1b39d8dd4ddc0a67aeea792e4438e6e1e968f138cf09999ce2d73acdf38ee155b4a9479b76d6801c57aea5ca7b4231855b3cf5d16859b81467e45b2c3f4b424e5642a2d2b6e29be9d96955916f7d16f9a29f989a5c1a3479770fba5fd51ce7d70555ff48a57766029268904b524ecb1eeb4a8c146aa0cc9a02575355c2cc37fcb439d1244d799aa2ce2896d10e29d4c396c1e40a07854e1582e22b4a282de1885a03ec3601ae4efb7ac5a9112804f4c96bb013264f2f1351772054e951b885b85be08e9cc3bd3984f6db0df4624ec290fb98fc3ae6b453df10497811f3df24fa0baa4859d2ac1e1cf03de290c68966084341b9344ac239ba40296f5b24b6d8cf6107c2587b3250e96b5eb19fa6507bcf68d1dc720a1352345816a06c41c22774123cb0b74ce127cd65e7be0ae65ff7855c0de29657b16ec9ff3078dc11381fab8b425f5f5bdc990878709fd76b30be115f3c871a4d6713745cf6279bea8453cfe7aa8de7a6e06f5f12c0b8f9c0a1264ca1526bb12b2eb040cbd40ec96e1d3f68136883a677c999724c4313d528dcd78179d8aa21b0b86985fad91607d827ffcb6e4890d0a8669dca1dbd0f4a622a640affbfa7328524e2e78c69395d08f10a9121a57baa07cc3423d3311855081bf8874976820a5e91d9ff815aabd9f89feb90421e7bd412805b6edfd27b01afe3fa57b4136ccbfdd8675600499bfdd17dc18bdd04140dc974bd664a8ba403b5a8311c27c536ea588050ac1508f6876b82b1b77e839989096c7232e7f9082d8748dd29ce786064382d4e1eba374f29889e84224680442685466cfd7ae5f552a1a989df1efeca550880e626619998d4b2279441b893dac3038d6920273f4e387dc54c480093efc527e236195615300f0fad3bfe69ceca17279b5b2b844fda9cc73d81ed0c40c5af71681948de514f1cb72534fb04d0663f6b03682bb8000907c2962df193263399b7b3950b759f61113ec44e551d79b69c533f9d0aeeed531c862de793da36f10a64595293022fe9110516a52e5a6dd1be540e424518fdb6416f113ddba8aadbeaf8d9bea61619f682da1c2930706bc2fab0559af7f3593809fe24c2db4c91e44b263dc72f08fafeb2ffab556cd96219c808b56e955c78e2f1811c2d76e0a2ca37a4a9c9028d4fca4c06f08626d16e896a217405b2288451b7e229e98007b8f73398954a993de9abbfe7ede20c5bcec9c30c237ea0d093e9e33636cb49d1267e3f9bfd558fcf9fa27d1052403204a4d767185e68a278d3
```

When I run this against 9239ac3312a6689f272ab417b3517d6eb924a800, I'm getting the output of modexp, but I would guess it should return an error.

**Comment by krlosMata** [comment]
`modexp` exceeding the 8192 limit will revert with no GAS consumption: https://github.com/0xPolygonHermez/zkevm-rom/blob/v9.0.0-rc.1-fork.13/main/precompiled/end.zkasm#L26

Very similar when an unsupported precompiled is called: https://github.com/0xPolygonHermez/zkevm-rom/blob/v9.0.0-rc.1-fork.13/main/precompiled/revert-precompiled.zkasm#L1C1-L1C18

**Comment by hexoscott** [comment]
Fixed by #1391
